### DISCUSSION TITLE
Error "The information you’re about to submit is not secure" when trying to login

### DISCUSSION THREAD

**Discussion by brockhager** [fisharebest/webtrees#4995] [Q&A] [answered]
I get the error "The information you’re about to submit is not secure" when trying to login.   

I have tried multiple browsers and I get the same error. 

![capture](https://github.com/fisharebest/webtrees/assets/8868577/f60b1800-6a57-4fc5-b323-cd940c408566)

I was going to make an update to .htaccess file,  but thought I would check here first.

**Comment by kiwi3685** [comment]
Thats a standard browser response when trying to access a web site that either does not user a secure "HTTPS" URL or it does and the certificate is expired.

What's the url of your site?

**Reply by brockhager** [reply]
https://getblockchain.tech/webtrees/index.php

I have a good SSL cert installed on my website.  There is something wrong with the installation or I still need to configure something.  I did an install using Installertron and my hosting is with Godaddy.  I have PHP version 8.0

**Reply by kiwi3685** [reply]
OK. I don't see the error you do, and I agree, you have a valid SSL cert.

However, there are a lot of these errors in the dev tools:

`[blocked] The page at https://getblockchain.tech/webtrees/index.php?route=%2Fwebtrees%2Flogin&url= was not allowed to run insecure content from http://getblockchain.tech/webtrees/public/css/vendor.min.css?v=2.1.20.
`
And also this:

```
Failed to load resource: Cross-origin redirection to
https://getblockchain.tech/webtrees/index.php?
route=%2Fwebtrees%2Fwebmanifest.json denied by Cross-
Origin Resource Sharing policy: Origin https://getblockchain.tech is not allowed by Access-Control-Allow-Origin. Status code: 301

```
I'd say the .htaccess file is definitely wrong somehow.

**Reply by brockhager** [reply]
Sorry - I had some code in the .htaccess file from some testing I was doing.  I have removed it.  Can you check one more time to see if you see the same errors now?

**Reply by kiwi3685** [reply]
I see the website OK, but still some errors:

<img width="1436" alt="Screenshot 2024-06-12 at 12 10 25 PM" src="https://github.com/fisharebest/webtrees/assets/1231773/8227bbde-c304-4215-94ca-76f9971b47e6">

**Reply by kiwi3685** [reply]
Check your webtrees/data/config.ini.php file to see if you have the correct base_url set there.

**Reply by kiwi3685** [reply]
Or, it could be this issue:

[Use base_url in webmanifest.json #4966](https://github.com/fisharebest/webtrees/issues/4966)

**Comment by fisharebest** [comment, marked as answer]
> Check your webtrees/data/config.ini.php file to see if you have the correct base_url set there.

You would get this error if the `base_url` contained `http://` when the site itself is `https://`.

**Reply by brockhager** [reply]
This is what was the issue.   I changed HTTP to HTTPS and now the site works!   Thanks guys!!
